src/main/scala/scalafix/internal/sbt/ScalafixInterface.scala

```diff
@@ -105,11 +107,33 @@ class ScalafixInterface private (
       try arg(acc)
       catch { case NonFatal(e) => throw new InvalidArgument(e.getMessage) }
     }
-    new ScalafixInterface(newScalafixArguments, this.args ++ args)
+    new ScalafixInterface(
+      newScalafixArguments,
+      postScalafixHooks,
+      this.args ++ args
+    )
   }
 
   def run(): Seq[ScalafixError] =
-    scalafixArguments.run().toSeq
+    try {
+      scalafixArguments.run().toSeq
+    } finally {
```

Comment:
it might be possible to be more precise (run this only on `LinterError`), but I decided to be safe and trigger the callback whenever this runs, because mutations are applied after each rule so there is no guarantee that nothing was touched when we get back an exception